version.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
Should the commiter run this before pushing the release tag?
If so, would be nice to add that info to the readme.

Reply:
This script does add the version tag to the repository. I agree it would be useful to have it described in readme